M/MuJoCo/patches/mingw.patch

```diff
@@ -0,0 +1,149 @@
+From ea76e286c041a705250a1018e6937b09d9d40505 Mon Sep 17 00:00:00 2001
+From: Jamie Mair <JamieMair@users.noreply.github.com>
+Date: Sat, 19 Aug 2023 11:29:51 +0100
+Subject: [PATCH 1/1] mingw patch
+
+---
+ CMakeLists.txt                    |  8 ++++----
+ cmake/MujocoOptions.cmake         |  1 -
+ dist/mujoco.rc                    |  2 +-
+ dist/simulate.rc                  |  2 +-
+ src/engine/engine_crossplatform.h | 10 +++++-----
+ src/engine/engine_util_errmem.c   |  8 +++++---
+ test/fixture.cc                   |  2 +-
+ 7 files changed, 17 insertions(+), 16 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2f04d7f..f6ce7c2 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -38,10 +38,10 @@ enable_language(CXX)
+ 
+ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
+ 
+-option(MUJOCO_BUILD_EXAMPLES "Build samples for MuJoCo" ON)
+-option(MUJOCO_BUILD_SIMULATE "Build simulate library for MuJoCo" ON)
+-option(MUJOCO_BUILD_TESTS "Build tests for MuJoCo" ON)
+-option(MUJOCO_TEST_PYTHON_UTIL "Build and test utility libraries for Python bindings" ON)
++option(MUJOCO_BUILD_EXAMPLES "Build samples for MuJoCo" OFF)
```

Comment:
I wonder why these options are patched to "off" instead of just setting them to "off" via command line?

Reply:
I have switched to the command line version.